libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/timeline/item/event/OtherState.kt

```diff
@@ -33,7 +33,7 @@ sealed interface OtherState {
     data object RoomJoinRules : OtherState
     data class RoomName(val name: String?) : OtherState
     data object RoomPinnedEvents : OtherState
-    data class RoomPowerLevels(val users: Map<String, Long>) : OtherState
+    data class RoomUserPowerLevels(val users: Map<String, Long>) : OtherState
```

Comment:
For my understanding, the keys of `users` are userId?

Reply:
Yes, they're user ids mapped to custom power levels, to distinguish them from the default one in `user_default`.